support/vm/vm.go

```diff
@@ -33,7 +33,7 @@ type VM struct {
 	currentEpoch   abi.ChainEpoch
 	networkVersion network.Version
 
-	actorImpls  ActorImplLookup
+	ActorImpls  ActorImplLookup
```

Comment:
Not important: This is a bit weird. I don't remember it (but maybe I didn't review the original).